# Generated for NaoQi method ALAudioDevice.enableEnergyComputation
# Enables the computation of the energy of each microphone signal

---


